.release-please-manifest.json

```
@@ -8,6 +8,6 @@
   "images/strimzi-kafka-connect": "1.7.0",
   "images/hive-metastore": "1.2.0",
   "images/warehousekeeper": "0.1.5",
-  "images/dsf-bpe-full": "1.4.1",
+  "images/dsf-bpe-full": "1.4.2",
```

```suggestion
  "images/dsf-bpe-full": "1.4.1",
```

No need for this, it's updated automatically based on your commit message!